plugin/al/usdtransaction/AL/usd/transaction/CMakeLists.txt

```diff
@@ -85,6 +85,25 @@ set_target_properties(${USDTRANSACTION_PYTHON_LIBRARY_NAME}
     LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 )
 
+# set the RPATH for _AL_USDTransaction
+if(IS_MACOSX OR IS_LINUX)
+    mayaUsd_init_rpath(rpath "${PROJECT_NAME}")
+    mayaUsd_add_rpath(rpath "../../../..")
+    mayaUsd_install_rpath(rpath ${USDTRANSACTION_PYTHON_LIBRARY_NAME})
+endif()
```

Comment:
Set the rpath for _AL_USDTransaction which has a dependency on libAL_USDTransaction.dylib